portfolio/src/main/webapp/style.css

```diff
@@ -1,5 +1,5 @@
 body {
-    background-color:rgb(198, 245, 245);
+    #background-color:rgb(198, 245, 245);
```

Comment:
Is this change intended?

Reply:
I am not sure why we added the hashtag before background-color, could you clarify?